src/main/java/org/nest/utils/AstUtils.java

```diff
@@ -353,7 +354,7 @@ public static String getNameOfDerivedVariable(ASTShape astShape) {
 
   public static String convertDevrivativeNameToSimpleName(final ASTVariable astVariable) {

```

Comment:
Do we need this function at all? I mean, the `toString` has the same functionality.